src/OTSubscriber.js

```diff
@@ -21,15 +22,18 @@ export default class OTSubscriber extends Component {
     this.componentEventsArray = Object.values(this.componentEvents);
     this.otrnEventHandler = getOtrnErrorEventHandler(this.props.eventHandlers);
   }
-  componentWillMount() {
-    const { sessionId, eventHandlers } = this.props;
-    this.streamCreated = nativeEvents.addListener(`${sessionId}:${this.componentEvents.streamCreated}`,
-      stream => this.streamCreatedHandler(stream));
-    this.streamDestroyed = nativeEvents.addListener(`${sessionId}:${this.componentEvents.streamDestroyed}`,
-      stream => this.streamDestroyedHandler(stream));
-    const subscriberEvents = sanitizeSubscriberEvents(eventHandlers);
-    OT.setJSComponentEvents(this.componentEventsArray);
-    setNativeEvents(subscriberEvents);
+  componentDidMount() {
```

Comment:
Shouldn't we put all the event listeners in the constructor (like you did on the Publisher file)?